hrms/tests/test_utils.py

```diff
@@ -92,3 +93,25 @@ def create_department(name: str, company: str = "_Test Company") -> str:
 	department.update({"doctype": "Department", "department_name": name, "company": "_Test Company"})
 	department.insert()
 	return department.name
+
+
+def create_job_applicant(**args):
```

Comment:
+1 on creating this util

Reply:
Um, actually I didn't create it; I just moved it from `test_job_applicant.py` 😅